nixos/modules/services/databases/couchdb.nix

```diff
@@ -4,20 +4,29 @@ with lib;
 
 let
   cfg = config.services.couchdb;
-  configFile = pkgs.writeText "couchdb.ini"
+  useVersion2 = strings.hasPrefix "2" (builtins.parseDrvName cfg.package.name).version;
```

Comment:
Using versionAtLeast would include any later versiony, not only 2.x builds.  Is that a problem?